README.md

```diff
@@ -39,7 +39,8 @@ This repo provides information and issue tracking for the following NuGet packag
 | [Microsoft.UI.Xaml.Core.Direct](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct) | Low-level APIs that can enable better performance when creating middleware components. Not intended or required for general application use |
 
 ## Supported OS versions
-* Windows Insider Preview (17135+)
+* Windows Insider Preview (17764+)
```

Comment:
❓ The changelist description mentioned updating to include 17763, but you also are bumping up to a new insider preview version number. Given that this says "+" is it necessary to keep incrementing these with each new preview build? Or just opportunistically when the file is touched?

Reply:
Good question. I'm fine with either leaving it out or just saying that we support latest Windows Insider Preview with no version. Or are you suggesting that we just not have a specific version list at all? @jesbis, thoughts?